misc/keyvaluestorage/src/node-js/index.ts

```diff
@@ -1,62 +1,62 @@
-import Database, { Database as IDatabase } from "better-sqlite3";
+
 import { safeJsonParse, safeJsonStringify } from "safe-json-utils";
 
 import {
   IKeyValueStorage,
   KeyValueStorageOptions,
-  parseEntry,
 } from "../shared";
-import { Statements } from "./sqlite";
+import db from "./level_db"
 
-const DB_NAME = "walletconnect_kvs.db";
-const TABLE_NAME = "keyvaluestorage";
+const DB_NAME = "walletconnect_level.db";
 
 export class KeyValueStorage implements IKeyValueStorage {
-  private readonly database: IDatabase;
-  private readonly statements: Statements;
+  private database;
 
   constructor(opts?: KeyValueStorageOptions) {
-    this.database = new Database(opts?.database || DB_NAME);
-    this.statements = new Statements(opts?.table || TABLE_NAME);
-    this.database.prepare(this.statements.createTable()).run();
+    this.database = db.create(opts?.database || opts?.table || `${DB_NAME}`)
```

Comment:
So in terms of top-level API, we're now making the `database` and `table` opts interchangeable here right?

Reply:
Yes, In this implementation they are interchangeable